tjreports/site/views/reports/tmpl/default.php

```diff
@@ -65,6 +64,12 @@
 	$document->addScriptDeclaration("var csv_export_inprogress='{$message['inprogress']}';");
 	$document->addScriptDeclaration("var tj_csv_site_root='{$siteUrl}';");
 }
+
+if ($this->showSummaryReport == 'Yes')
+{
+	HTMLHelper::_('script', 'https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.3/Chart.bundle.min.js');
```

Comment:
@pranotiTechjoomla load this on the same location where we have added script. Also don't use the CDN url